src/SmartComponents/Remediations/RemediationWizard.js

```diff
@@ -207,7 +209,10 @@ class RemediationWizard extends Component {
     /*
      * State manipulation
      */
-    onNameChange = name => this.setState({ name });
+    onNameChange = name => this.setState({
+        name,
+        isValidated: (name.length ? true : false)
```

Comment:
@ryelo as discussed name should not be required

Reply:
@jharting fixed!